src/content/docs/fr/2/guide/frontend/nextjs.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Next.js
+i18nReady: true
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+Next.js est un framework de React. Apprenez-en plus au sujet de Next.js sur https://nextjs.org. Ce guide est valable à partir de Next.js 13.4.19.
+
+## Liste des Vérifications
+
+- Utilisez des exports statiques en définissant `output: 'export'`. Tauri ne supporte pas les solutions basées sur serveur.
+- Utilisez `internal-ip` pour une compatibilité sur mobile pour pouvoir configurer `assetPrefix`. Ceci est requis pour que le serveur résolve correctement les assets.
+- Utilisez `out/` en tant que `distDir` dans `tauri.conf.json`.
+
+## Configuration d'Exemple

```

Comment:
```suggestion
## Exemple de configuration

```

Reply:
We can keep `Configuration d'exemple` but we need to lowercase Exemple

Reply:
> We can keep `Configuration d'exemple` but we need to lowercase Exemple here. It's common in English to capitalize every word in title, not in French.

Oh okay i didn't thought it was an english only thing